BreadCrumb Control.xcodeproj/project.pbxproj

```diff
@@ -605,10 +609,11 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_CODE_COVERAGE = NO;
 				INFOPLIST_FILE = "BreadCrumb Control/Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.4;
```

Comment:
Thank you for your PR!!

I think, deployment target 12.4 is too new.  In `8.0` case, it has something problem?
Even if we update it, I think `10.0` is good.

@apparition47 How do you think?

Reply:
Thank you for your reply.
I should specify about 10.0 for deployment so that even a older version can be used without problems.
If you have target you want, follow your instructions.

Reply:
`10.0` is a safe target. Users targetting `8.0` can use an older version of this Pod. Thanks!